CHANGELOG.md

```diff
@@ -1,5 +1,26 @@
 This file contains a list of changes, new features, and fixes in each release of Monite SDK.
 
+## @monite/sdk-react@3.13.0 & @monite/sdk-api@3.13.0
+This package is a minor release with no breaking changes.
+
+### New features and improvements
+- Added by counterpart name and status to the `ReceivablesTable` component.
+- Improved overall layout, security, and performance of the React SDK.
+
+### Bug fixes
+- Fix layouts of PDF toolbars that caused scroll issues on PDFs.
+
+## @monite/sdk-react@3.12.0 & @monite/sdk-api@3.12.0
+This package is a minor release with no breaking changes.
+
+### New features and improvements
+- Added the `InvoiceStatusChip` and `PayableStatusChip` components to show the status of invoices and payables.
+- Introduced the `TablePagination` to handle customization and pagination on all table components.
+- Added the ability to edit an invoice on the `InvoiceDetails` component.
+- Introduced the `currencyNumberFormat` field on the locale prop that handles currency display on the SDK.
+ - Improved context management and usability of the `MoniteProvider` wrapper.
```

Comment:
extra space at the very start of the string

Reply:
I could not find a related task in the release in Jira. Did we actually implement something like that?

Reply:
> I could not find a related task in the release in Jira. Did we actually implement something like that?

Which list item do you refer to? If you mean the point about the `MoniteProvider`, I kind of got it from the release announcement you posted:
> [d180bf4](https://github.com/team-monite/monite-sdk/commit/d180bf4d3646d81056157d5032ccd1141d109554): 1. Providers that may affect the customer’s context have been moved to <MoniteScopedProviders/>. 2. All exported components and all components opening in <Portal/> receive Monite-ContainerCssBaseline class name. It provides styles for the color scheme of native controls (dark/light) and default colors for components. > Backgrounds will not be added (neither for dark nor light themes), to minimize conflicts with embedding in customer applications. 3. The <MoniteContext/> interface has been updated to include:
theme: Moved from <MoniteThemeContext/>.
i18n: Added instead of code, pre-created via Suspense to contain all necessary data for components and hooks.
dateFnsLocale: Added for DatePicker functionality, pre-created via Suspense.
queryClient: Added for usage within <MoniteScopedProviders/>.
These changes improve the modularity and maintainability of the MoniteProvider while providing enhanced context management and usability for Monite SDK customers.


Do you think it should be worded differently?